app/views/charges/_charge.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= form_tag charges_path do %>
+  <%= hidden_field_tag :story_id, @story.id %>
+  <article>
+    <% if flash[:error].present? %>
+      <div id="error_explanation">
+        <p><%= flash[:error] %></p>
+      </div>
+    <% end %>
+    <label class="amount">
+      <span>Amount: <%= number_to_currency(@price) %></span>
+    </label>
+  </article>
+
+  <script class='stripe-button' src="https://checkout.stripe.com/checkout.js" 
+  data-key="<%= Rails.configuration.stripe[:publishable_key] %>" 
+  data-amount="<%= @story.price.to_i * 100 %>" 
```

Comment:
Do you mean put the rails variables in the controller or the entire <script> itself?